<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>650</y>
      <width>711</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="reconstructionGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Reconstruction Parameters</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="reconstructionFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="smoothingLabel">
        <property name="text">
         <string>Smoothing Length:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="isoLabel">
        <property name="text">
         <string>ISO-Value:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="radiusLabel">
        <property name="text">
         <string>Support Radius:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="gridCellsLabel">
        <property name="text">
         <string># of grid cells per axis:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="smoothingDoubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>0.028000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="isoDoubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.400000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="supportDoubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>0.056000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="gridCellsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>65</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="postProcessingGroupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>530</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Post-Processing</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="postSmoothingLabel">
        <property name="text">
         <string># of Smoothings</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="postSmoothingSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="postSmoothingPushButton">
        <property name="text">
         <string>Smooth</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="simplifyPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Simplify Surface</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="simulationMetricsGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulation Metrics</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>16</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="simulationFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="numberParticlesLabel">
        <property name="text">
         <string># of Particles:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="numberParticlesLabel_2">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="viewportGroupBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>661</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Viewport Options</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>611</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="viewportGridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="surfaceCheckBox">
        <property name="text">
         <string>Surface</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="transparencySlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="transparencySpinBox">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="particlesCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Particles</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="transparencyCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Transparency</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>901</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="topHorizontalLayout">
     <item>
      <widget class="QLabel" name="exportFormatLabel">
       <property name="text">
        <string>Export Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="exportFormatComboBox">
       <item>
        <property name="text">
         <string>.ply</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.vtk</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sphLabel">
       <property name="text">
        <string>SPH Interpolation kernel:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sphComboBox">
       <item>
        <property name="text">
         <string>Cubic Spline</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spiky</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Option 3</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="topInnerHorizontalLayout">
       <item>
        <widget class="QLabel" name="ompNumThreadsLabel">
         <property name="text">
          <string># of OpenMP Threads:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ompNumThreadsSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="batchJobPushButton">
       <property name="text">
        <string>Generate Batch Job...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reconstructPushButton">
       <property name="text">
        <string>Reconstruct...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Henrik Müller/Pictures/Play-Icon.png</normaloff>../../Henrik Müller/Pictures/Play-Icon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeView" name="fileSelectTreeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>640</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="bottomHorizontalLayout">
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>elapsed Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ramLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ramLabel_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="exportPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>143</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Directory...</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>49</y>
      <width>881</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="viewportLayout"/>
   </widget>
   <widget class="QPushButton" name="loadPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load selected Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetCamPushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>520</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextFramePushButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>520</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevFramePushButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>520</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous Frame</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>transparencySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>transparencySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>810</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>1207</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transparencySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>transparencySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1198</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>1024</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
